apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/CreateOrder.tsx

```diff
@@ -220,7 +227,12 @@ export const CreateOrder = () => {
 
             <TopTokenActions />
 
-            <EditSellAmount error={sellError} />
+            <EditSellAmount
```

Comment:
`EditSellAmount` can change the `tokenId` and the `quantity`, if this one is just for quantity, add it to the API, `sellInputQuantity`, same for the hook `useSellQuantityInput`